image-search/Models/Alert.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Alert.swift
+//  image-search
+//
+//  Created by Renato Vieira on 22/10/19.
+//  Copyright © 2019 Stant. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class Alert: NSObject {
+
+    func display(_ controller: UIViewController, _ titulo: String, _ mensagem: String) {
```

Comment:
Oh mannnnn!  "titulo" no! hauhauhuahuah (title and message).

Reply:
kkkkkkkkkkkkkkk forgive me!!!!:)
Done